department-of-veterans-affairs/caseflow
# Fixed the page flow when closing ad hearing day modal

Resolves [CASEFLOW-1014](https://vajira.max.gov/browse/CASEFLOW-1014)

### Description
Adds a function to set the focus back to the button that opened the dialog so that it can be re-focused when the dialog is closed

### Acceptance Criteria
- [ ] Meaning is preserved for screen readers when closing the add hearing date dialog

### Testing Plan
1. Sign in as user `BVASYELLOW` http://localhost:3000/test/users
2. Navigate to the hearings schedule: http://localhost:3000/hearings/schedule
3. Click the "Add Hearing Date" link under the date range search
4. Click the "Go Back" button when the dialog opens
5. Use the tab keys and make sure that you navigate to the first column of the table
